internal/dockerutil/setup.go

```diff
@@ -51,29 +52,57 @@ func DockerSetup(t *testing.T) (*dockertest.Pool, string) {
 }
 
 // dockerCleanup will clean up Docker containers, networks, and the other various config files generated in testing
-func dockerCleanup(testName string, pool *dockertest.Pool) func() {
+func dockerCleanup(t *testing.T, pool *dockertest.Pool) func() {
 	return func() {
 		cont, _ := pool.Client.ListContainers(docker.ListContainersOptions{All: true})
 		for _, c := range cont {
 			for k, v := range c.Labels {
```

Comment:
`c.Labels` should be a `map[string]string`, so I think we can directly check `v, ok := c.Labels[CleanupLabel]`.